pkg/broker/broker.go

```diff
@@ -35,46 +36,61 @@ import (
 
 const registrySecretName = "registry-secret"
 
+//go:generate counterfeiter ./ ClusterFactory
+type ClusterFactory interface {
```

Comment:
Is this the right place for this? I would expect it in the `k8s` package. Also, the generated doubles aren't check in, so ci would fail.